src/pages/index.js

```diff
@@ -1,192 +1,224 @@
         voluptatum. Esse vel est debitis. Natus assumenda laborum tempore et
         inventore. Doloribus expedita reprehenderit.
-      </Typography>
-
-      <Button
-        variant="contained"
-        color="secondary"
-        component={Link}
-        to="/quizz"
-        lang={i18n.language}
-      >
-        {t('See mesures')}
-      </Button>
-
-      <Divider />
-
+        </Typography>
+        <Grid container className={classes.module__button} justify="center">
+          <Button
+            variant="contained"
+            color="secondary"
+            component={Link}
+            to="/quizz"
+            lang={i18n.language}
+          >
+            {t('See mesures')}
+          </Button>
+        </Grid>
+      </Grid>
       {/* Summary')}</ */}
+      <Grid container justify="center" className={classes.module}>
+        <Typography className={classes.module__title} variant="h1" gutterBottom>{t('Summary')}</Typography>
 
-      <Typography variant="h2" gutterBottom>{t('Summary')}</Typography>
-
-      <Typography variant="body1" paragraph>
+        <Typography variant="body1" paragraph>
         Qui consequatur sit nesciunt omnis voluptatem consequatur error ut
         placeat. Rerum accusantium sed est at aut rerum. Totam rem accusamus
         consequatur assumenda quo voluptatem dolorem excepturi. Ut voluptas eos
         voluptas laboriosam iure laboriosam qui. Est sed libero quia ut magni
         vitae aut minus.
-      </Typography>
+        </Typography>
 
-      <Typography variant="body1" paragraph>
+        <Typography variant="body1" paragraph>
         Quod totam consequatur incidunt pariatur. Eum et sed quasi cumque
         tempore. Totam modi cupiditate pariatur aspernatur soluta qui. Minus
         quae vitae fugiat.
-      </Typography>
+        </Typography>
 
-      <Typography variant="body1" paragraph>
+        <Typography variant="body1" paragraph>
         Ut rerum sed a ipsam ut ut voluptates quibusdam voluptas. Consequuntur
         minus quis. Non placeat dolor aut eos a officiis sunt omnis.
-      </Typography>
-
-      <Button
-        variant="contained"
-        color="secondary"
-        component={Link}
-        to="/quizz"
-        lang={i18n.language}
-      >
-        {t('Start quizz')}
-      </Button>
+        </Typography>
+        <Typography variant="h2">Agriadapt roadmap for adaptation </Typography>
+        <img src={`../images/roadmaps/roadmap-${i18n.language}.svg`} alt={t('Agriadapt roadmap for adaptation')} />
+        <Grid container className={classes.module__button} justify="center">
+          <Button
+            variant="contained"
+            color="secondary"
+            component={Link}
+            to="/quizz"
+            lang={i18n.language}
+          >
+            {t('Start quiz')}
+          </Button>
+        </Grid>
+      </Grid>
     </Layout>
   );
 };
 
+Layout.propTypes = {
+  theme: PropTypes.node.isRequired,
```

Comment:
Theme should not be provided as props (neither required)

Reply:
done